### ISSUE TITLE
Why loading pre-trained models does not match？

### ISSUE THREAD

**Issue by fadaishaitaiyang** [ViTAE-Transformer/ViTDet#15]
![e68847d37da998b0bda6d0fd406fc0b](https://user-images.githubusercontent.com/63134541/173710514-5d45e5b2-1c29-4b49-a2a6-d32e508c1721.jpg)

**Comment by Annbless** [comment]
Hi, 

It seems that you have modified the backbone structures with convolution and batch normalization in the FFN part.

**Comment by fadaishaitaiyang** [comment]
I just modified the backbone structures with convolution in vit part.
